### DISCUSSION TITLE
Can't export ipynb file to Pdf

### DISCUSSION THREAD

**Discussion by mhamza6051** [AammarTufail/python_ka_chilla2023#19] [Q&A]
i can't export ipynb file to pdf, i had also download nbconvert, pandoc etc but can't export to pdf

**Comment by mhamza6051** [comment]
![image](https://user-images.githubusercontent.com/121826823/210864735-aea62dd8-ca20-4ffa-bb62-259cc4f6c125.png)

**Reply by SahilJatoi744** [reply]
I would suggest you to download other extension which can convert .ipynb to pdf or you can use online platforms aswell
https://www.vertopal.com/en/convert/ipynb-to-pdf
